socorro/unittest/external/elasticsearch/test_supersearch.py

```diff
@@ -1533,6 +1533,48 @@ def test_get_with_pagination(self):
         eq_(res['total'], 21)
         eq_(len(res['hits']), 0)
 
+    def test_get_with_sorting(self):
```

Comment:
This test works against both 0.90 and 1.x?  i.e. Does it need a `@maximum_es_version()`?